CHANGELOG.md

```diff
@@ -10,21 +10,28 @@ We would recommend to consult log of the
 [DataLad git repository](http://github.com/datalad/datalad) for more details.
 
 
-## 0.12.0rc2 (??? ??, 2019) -- will be better than ever
+## 0.12.0rc2 (Mar 18, 2019) -- revolution!
 
-bet we will fix some bugs and make a world even a better place.
+### Fixes
 
-### Major refactoring and deprecations
+- `GitRepo.dirty` does not report on nested empty directories ([#3196]).
 
-- hopefully none
+- LORIS token authenticator now works with Python 3.
 
-### Fixes
+- Introduce an explicit switch to enable decoding of fetched content
+  in `BaseDownloader`
```

Comment:
The LORIS and BaseDownloader entries will already be covered by 0.11.4 (#3228), so I don't think we need them here.

Reply:
OK, thx for the info. Will kill them later, don't want to go through another test run, but rather proceed with the revolution partner release. Hope that is fine with all.

Reply:
Works for me.  Thanks.